src/sentry/models/organization.py

```diff
@@ -129,6 +129,10 @@ class Organization(Model):
                 "disable_new_visibility_features",
                 "Temporarily opt out of new visibility features and ui",
             ),
+            (
+                "demo_mode",
+                "Mark an organization as a demo org so it will be destroyed 24 hours after being created.",
```

Comment:
I would leave out the "Will be destroyed 24 hours after..." since that number is subject to change and you don't want to have to remember to come back here"

I think just saying

> Marks this organization as a demo org